packs/feats/call-the-hurricane.json

```diff
@@ -11,7 +11,7 @@
         },
         "category": "class",
         "description": {
-            "value": "<p>Massive waves spiral around you, with you as the eye of the hurricane. The waves appear in a @Template[type:emanation|distance:20], or a @Template[type:emanation|distance:30] if you're in a body of water. Each creature in the area takes @Damage[6d8[bludgeoning]] damage with a @Check[type:reflex|dc:resolve(@actor.system.attributes.classDC.value)|basic:true] save against your class DC. A creature that fails its save is battered by the waves and pushed 10 feet (or 20 feet on a critical failure).</p>\n<hr />\n<p><strong>Level (+2)</strong> The damage increases by [[/r 1d8]].</p>"
+            "value": "<p>Massive waves spiral around you, with you as the eye of the hurricane. The waves appear in a @Template[type:emanation|distance:20], or a @Template[type:emanation|distance:30] if you're in a body of water. Each creature in the area takes @Damage[(floor((@actor.level - 8)/2)+6)d8[bludgeoning]] damage with a @Check[type:reflex|dc:resolve(@actor.system.attributes.classDC.value)|basic:true] save against your class DC. A creature that fails its save is battered by the waves and pushed 10 feet (or 20 feet on a critical failure).</p>\n<hr />\n<p><strong>Level (+2)</strong> The damage increases by 1d8.</p>"
```

Comment:
`@Damage[(floor((@actor.level - 8)/2)+6)d8[bludgeoning]]` resolves to 2d8 when rendered in compendium (it assumes the actor level is 1). We can work around this by using the `max()` function. Two choices here are to just wrap the whole thing in that and feed it the minimum dice `@Damage[(max(6,floor((@actor.level - 8)/2)+6))d8[bludgeoning]]` but we can also use this directly on the actor level call to make sure the minimum number that resolves it 6d8 `@Damage[(floor((max(8,@actor.level)-8)/2)+6)d8[bludgeoning]]`

Unfortunately that's going to apply to most of these :(

Reply:
I'm indifferent to whatever approach we take, the first is simpler the last is maybe cleaner looking but these formulas also tend to be hard to parse by hand anyways.

Reply:
Added those changes, opted for the second one.